pages/[filterName]/insights/index.tsx

```diff
@@ -74,6 +77,35 @@ const InsightsHub: WithPageLayout = () => {
           Create a new Insight Page
 
       </Link>
+
+      <div className="py-1 md:py-4 flex w-full md:mt-5 justify-between items-center">
+        <div>
+          <div className="">
+            <PaginationResults
+              from={page === 1 ? (insightsMeta.itemCount > 0 ? page : 0) : (page - 1) * insightsMeta.limit + 1}
+              to={page * insightsMeta.limit <= insightsMeta.itemCount ? page * insightsMeta.limit : insightsMeta.itemCount}
+              total={insightsMeta.itemCount}
+              entity={"insights"}
+            />
+          </div>
+        </div>
+        <div>
+          <div className="flex items-center gap-4">
+            <Pagination
+              pages={[]}
```

Comment:
what would pages be here?

Reply:
The total number of pages. 
- empty array
![Screenshot from 2023-02-16 20-57-23](https://user-images.githubusercontent.com/79809121/219485311-79286df0-4af5-4673-864c-a936ddf5e3fe.png)
- with the number of pages
![Screenshot from 2023-02-16 20-57-48](https://user-images.githubusercontent.com/79809121/219485395-e2bbe70c-9e3d-4c3e-a0b7-d88d465cca1a.png)